site/src/main/scala/scalatex/site/Styles.scala

```diff
@@ -1,9 +1,9 @@
 package scalatex.site
 import scalatags.Text.all._
-import scalatags.stylesheet.{CascadingStyleSheet, Sheet}
+import scalatags.stylesheet.{CascadingStyleSheet, SourceClasses}
 
 object Styles{
-  val css = Sheet[Styles]
+  val css = new Styles {}
```

Comment:
I tried witch SourceClasses as well but I was not able to make it compile. Is it correct?